### ISSUE TITLE
A Curious Question about Prediction Quality

### ISSUE THREAD

**Issue by reshalfahsi** [microsoft/FLAML#214]
Hi there,

Is it possible to obtain prediction quality? Is there any computation under the hood inside this package to find the conformal prediction? If there weren't any could you please add it as the feature request?

**Comment by sonichi** [comment]
I'm making it a feature request. What package do you use today to get prediction quality for models like lightgbm or random forest?

**Comment by reshalfahsi** [comment]
Have you heard about `nonconformist` package? I currently use that one.

**Comment by sonichi** [comment]
It seems that you can first run flaml to do hyperparameter tuning and model selection, and then pass `flaml.model` to the package. Give that a try?

**Comment by reshalfahsi** [comment]
I haven't but maybe I will give it a try.

**Comment by reshalfahsi** [comment]
Hi @sonichi,

It seems the `nonconformist` package requires the user to train alongside the wrapper from the package. You have to wrap the model and do the magic. But I recommend that you should implement the algorithm by yourself because when I apply XGBoost model I have to tweak the package a little bit.

**Comment by sonichi** [comment]
Yes it uses a wrapper. I was thinking that you could take the tuned model from flaml, wrap it with the package and retrain. Is there any issue with this approach? What tweak is required for XGBoost?
